src/main/java/de/rub/nds/modifiablevariable/ModifiableVariable.java

```diff
@@ -64,6 +65,7 @@
 @XmlAccessorType(XmlAccessType.FIELD)
 public abstract class ModifiableVariable<E> implements Serializable {
 
+    @XmlTransient
     protected Boolean autoformat = null;
```

Comment:
I think we can remove autoformat all together since we removed pretty printing

Reply:
done